Ubika/connector_ubika_cloud_protector_alerts.json

```
@@ -0,0 +1,46 @@
+{
+  "description": "",
+  "docker_parameters": "ubika_cloud_protector_alerts",
+  "name": "[BETA] Fetch new alerts from Ubika Cloud Protector",
+  "arguments": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "properties": {
+      "frequency": {
+        "type": "integer",
+        "description": "Batch frequency in seconds",
+        "default": 60
+      },
+      "chunk_size": {
+        "type": "integer",
+        "description": "The size of chunks for the batch processing",
+        "default": 10000
+      },
+      "intake_server": {
+        "description": "Server of the intake server (e.g. 'https://intake.sekoia.io')",
+        "default": "https://intake.sekoia.io",
+        "type": "string"
+      },
+      "intake_key": {
+        "description": "Intake key to use when sending events",
+        "type": "string"
+      },
+      "provider": {
+        "description": "Id of cirrus provider",
+        "type": "string"
+      },
+      "tenant": {
+        "description": "Id of cirrus tenant",
+        "type": "string"
+      },
+      "token": {
+        "description": "API token",
+        "type": "string"
+      }
+    },
+    "required": ["frequency", "intake_key"],
```

I think that `provider`, `tenant` and `token` are required. Not sure for `frequency`, as we have a default value.
```suggestion
    "required": ["provider", "tenant", "token", "intake_key"],
```